### ISSUE TITLE
Simplify the repository names...

### ISSUE THREAD

**Issue by SvenDowideit** [machine-drivers/discussion#9]
Atm, we have very long, and redundant names for the repos

Can we rename them to just `machine-drivers/qemu` etc?

@all

**Comment by SvenDowideit** [comment]
I'm going to see what it looks like with https://github.com/dhiltgen/docker-machine-kvm - see https://github.com/machine-drivers/kvm

**Comment by gbraad** [comment]
I noticed the naming... And unfortunate it has to be done by us. But
off-topic, don't think @ALL worked... Will check the teams setup later today

-- 

   Gerard Braad | http://gbraad.nl
   [ Doing Open Source Matters ]

**Comment by afbjorklund** [comment]
I prefer the long names. Otherwise I just end up confusing my directories.
e.g. git://git.qemu.org/qemu.git vs https://github.com/machine-drivers/docker-machine-driver-qemu.git

Or that clever "docker/machine" hack, that now just looks odd all by itself ?
It's even worse on GitHub, where I can't even create folders for my forks...

**Comment by SvenDowideit** [comment]
fair enough - I have such a huge number of projects and repos that I use a stack of subdirectories - and fo go, its all organised by `go get` :)

**Comment by gbraad** [comment]
similarly here. people can checkout in a directory like `git clone [repo]
[target]`. if this is part of the repo, most people will not notice.
but sure... it is an easy change... although it affects imports :-/

**Comment by SvenDowideit** [comment]
cool - I've renamed the kvm repo to https://github.com/machine-drivers/docker-machine-driver-libvirt
